### PR TITLE
docs: Update install.md

### PR DESCRIPTION
**Pull request by Chever-John**
I found A problem when installing `docker-compose`, and it was solved in time. I hope you can update the document.

### Related issues (optional)
Fixs #14367 and
#8445 in [github/docker/compose](https://github.com/docker/compose/issues/8445)

### CODE REVIEW

File: compose/install.md [hunk 1]
```diff
@@ -139,6 +139,15 @@ also included below.
     > with the version of Compose you want to use. For instructions on how to
     > install Compose `{{site.compose_v2_version}}` on Linux, see [Install
     > Compose 2.0.0 on Linux](../cli-command#install-on-linux)
+  
+    > Note: If you are using an operating system based on an arm architecture in 
+    > [MCU](https://en.wikipedia.org/wiki/Microcontroller), such as Raspberry Pi. 
+    > You should execute this command: 
+
+    ```console
+    $ sudo curl -L "https://github.com/docker/compose/releases/download/v{{site.compose_version}}/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```
Comment (thaJeztah) [inline compose/install.md:148]: This doesn't look correct for v1 installation; compose v1 uses tags without a `v` prefix (only compose v2 tags have a v prefix); were you  trying to use these instructions to install v2 of compose?

Comment (Chever-John) [reply compose/install.md:148]: Wait for a while, I will check it.

**Comment by usha-mandya** [comment]
Closing this PR as the `v` only works for Compose v1 installation. Feel free to continue the conversation after I close

**Comment by Chever-John** [comment]
> Closing this PR as the `v` only works for Compose v1 installation. Feel free to continue the conversation after I close

Ok, Tks!

**Pull request closed without merging**